ui/src/i18n/en.json

```diff
@@ -1,341 +1,346 @@
-        "settings": "Settings",
-        "version": "Version",
-        "theme": "Theme",
-        "personal": {
-            "name": "Personal",
-            "options": {
-                "theme": "Theme",
-                "language": "Language",
-                "defaultView": "Default View",
-                "desktop_notifications": "Desktop Notifications"
-            }
-        },
-        "albumList": "Albums",
-        "about": "About"
+    "user": {
+      "name": "User |||| Users",
+      "fields": {
+        "userName": "Username",
+        "isAdmin": "Is Admin",
+        "lastLoginAt": "Last Login At",
+        "updatedAt": "Updated at",
+        "name": "Name",
+        "password": "Password",
+        "createdAt": "Created at"
+      }
     },
     "player": {
-        "playListsText": "Play Queue",
-        "openText": "Open",
-        "closeText": "Close",
-        "notContentText": "No music",
-        "clickToPlayText": "Click to play",
-        "clickToPauseText": "Click to pause",
-        "nextTrackText": "Next track",
-        "previousTrackText": "Previous track",
-        "reloadText": "Reload",
-        "volumeText": "Volume",
-        "toggleLyricText": "Toggle lyric",
-        "toggleMiniModeText": "Minimize",
-        "destroyText": "Destroy",
-        "downloadText": "Download",
-        "removeAudioListsText": "Delete audio lists",
-        "clickToDeleteText": "Click to delete %{name}",
-        "emptyLyricText": "No lyric",
-        "playModeText": {
-            "order": "In order",
-            "orderLoop": "Repeat",
-            "singleLoop": "Repeat One",
-            "shufflePlay": "Shuffle"
-        }
+      "name": "Player |||| Players",
+      "fields": {
+        "name": "Name",
+        "transcodingId": "Transcoding",
+        "maxBitRate": "Max. Bit Rate",
+        "client": "Client",
+        "userName": "Username",
+        "lastSeen": "Last Seen At",
+        "reportRealPath": "Report Real Path"
+      }
+    },
+    "transcoding": {
+      "name": "Transcoding |||| Transcodings",
+      "fields": {
+        "name": "Name",
+        "targetFormat": "Target Format",
+        "defaultBitRate": "Default Bit Rate",
+        "command": "Command"
+      }
+    },
+    "playlist": {
+      "name": "Playlist |||| Playlists",
+      "fields": {
+        "name": "Name",
+        "duration": "Duration",
+        "owner": "Owner",
+        "public": "Public",
+        "updatedAt": "Updated at",
+        "createdAt": "Created at",
+        "songCount": "Songs",
+        "comment": "Comment",
+        "sync": "Auto-import",
+        "path": "Import from"
+      },
+      "actions": {
+        "selectPlaylist": "Select a playlist:",
+        "addNewPlaylist": "Create \"%{name}\"",
+        "export": "Export"
+      },
+      "message": {
+        "duplicate_song": "Add duplicate song ",
+        "song_exist": "There are duplicates being added to the playlist. Would you like to add the duplicates or skip them ?"
```

Comment:
Small nitpick: Can you please remove the space before the question mark?

Reply:
I have updated the message